pages/common/curl.md

```diff
@@ -3,34 +3,34 @@
 > Transfers data from or to a server.
 > Supports most protocols including HTTP, FTP, POP.
 
-- Download a URL to a file:
+- Download the contents of an URL to a file:
 
-`curl "{{URL}}" -o {{filename}}`
+`curl {{http://example.com}} -o {{filename}}`
 
-- Send form-encoded data:
+- Download a file saving it under the same name as indicated in the URL:
 
-`curl --data {{name=bob}} {{http://localhost/form}}`
+`curl -O {{http://example.com/filename}}`
 
-- Send JSON data:
+- Download a file, following [L]ocation redirects, and automatically [C]ontinuing/resuming a previous file transfer:
 
-`curl -X POST -H "Content-Type: application/json" -d {{'{"name":"bob"}'}} {{http://localhost/login}}`
+`curl -O -L -C - {{http://example.com/filename}}`
 
-- Specify an HTTP method:
+- POST application/x-www-form-urlencoded data:
 
-`curl -X {{DELETE}} {{http://localhost/item/123}}`
+`curl -d {{'name=bob'}} {{http://example.com/form}}`
 
-- Head request:
+- Send data, specifying a custom HTTP method, and including an extra header:
 
-`curl --head {{http://localhost}}`
+`curl -d {{'{"name":"bob"}'}} -X {{PUT}} -H {{'Content-Type: application/json'}} {{http://example.com/users/1234}}`
 
-- Include an extra header:
+- Head request:
```

Comment:
Let's just remove this one while we're at it, if we all agree it's redundant (I'll defer to @agnivade's comments on this).